redfish-core/lib/hypervisor_system.hpp

```diff
@@ -976,13 +957,8 @@ inline void handleHypervisorIPv6StaticPatch(
             return;
         }
 
-        if (errorInEntry)
-        {
-            return;
-        }
-
-        BMCWEB_LOG_ERROR(
-            "INFO: Static ip configuration request from client: {} - IP Address: {}, Gateway: {}, Prefix Length: {}",
+        BMCWEB_LOG_DEBUG(
```

Comment:
Please keep this as ERROR only. This was added for debugging dual HMC scenarios where we had hit some critical bugs at lab. Same comment for the other trace change as well.

Reply:
done